Source/WebCore/rendering/style/OffsetRotation.h

```diff
@@ -29,13 +29,18 @@
 
 namespace WebCore {
 
+struct BlendingContext;
+
 class OffsetRotation {
 public:
     OffsetRotation(bool hasAuto = false, float angle = 0);
 
     bool hasAuto() const { return m_hasAuto; }
     float angle() const { return m_angle; }
 
+    bool canBlend(const OffsetRotation&) const;
+    WEBCORE_EXPORT OffsetRotation blend(const OffsetRotation&, const BlendingContext&) const;
```

Comment:
I assume you plan to use this outside of Webcore in another patch

Reply:
Yes, although we could make it not exported for now and add the WEBCORE_EXPORT in due time.